File: backend/src/main/java/net/zorphy/backend/site/scotlandyard/dto/game/GameState.java
package net.zorphy.backend.site.scotlandyard.dto.game;

import net.zorphy.backend.site.core.http.dto.GameStateBase;
import net.zorphy.backend.site.scotlandyard.dto.graph.GraphNode;

import java.time.Instant;
import java.util.List;

public record GameState(
        Instant startTime,
        GameConfig gameConfig,
        List<GraphNode> map
) implements GameStateBase {
}

File: frontend/src/app/sites/seven-wonders/dto/game/GameState.ts
import {SavableGameState} from "../../../core/http/dto/SavableGameState";
import {PausableGameState} from "../../../core/http/dto/PausableGameState";
import {GameStateBase} from "../../../core/http/dto/GameStateBase";
import {GameConfig} from "./GameConfig";

export interface GameState extends GameStateBase, SavableGameState, PausableGameState {
  gameConfig: GameConfig;
}